app/customer.py

```diff
@@ -0,0 +1,11 @@
+class Customer:
+    def __init__(self, customer: dict) -> None:
+        self.name = customer["name"]
+        self.product_cart = customer["product_cart"]
+        self.milk = self.product_cart["milk"]
+        self.milk = self.product_cart["milk"]

```

Comment:
The `milk` attribute is being assigned twice. This is redundant and could lead to confusion. You should remove one of these assignments.